lib/crowbar/client/command/proposal.rb

```diff
@@ -22,10 +24,16 @@ module Proposal
 
         included do
           desc "Proposal specific commands"
-          command :proposal do |parent|
+          command "proposal" do |parent|
             parent.desc "Show a list of available proposals"
             parent.arg :barclamp
-            parent.command :list do |c|
+            parent.command "list" do |c|
+              c.desc "Format of the resulting output"
+              c.flag [:format], type: String, default_value: :table
+
+              c.desc "Filter output by criteria"
```

Comment:
Any better suggestion? Currently all listings can be filtered by any substring that matches with `include?`. Only the proposal show command is another case where you can define the path separated by a dot.